srcts/src/inputPolicies/inputBatchSender.ts

```diff
@@ -1,27 +1,21 @@
-import $ from "jquery";
-import type { EventPriority } from "./inputPolicy";
-import { InputPolicy } from "./inputPolicy";
+import type { InputPolicy, InputPolicyOpts } from "./inputPolicy";
 import type { ShinyApp } from "../shiny/shinyapp";
 
 // Schedules data to be sent to shinyapp at the next setTimeout(0).
 // Batches multiple input calls into one websocket message.
-class InputBatchSender extends InputPolicy {
+class InputBatchSender implements InputPolicy {
+  target!: InputPolicy; // We need this ...
   shinyapp: ShinyApp;
-  timerId: NodeJS.Timeout = null;
+  timerId: NodeJS.Timeout | null = null;
```

Comment:
```suggestion
  timerId: number | null = null;
```

Reply:
This should along with adding `"type": []` in the `compilerOptions` of `tsconfig.json`.